src/lualib/declarations/global.d.ts

```diff
@@ -1,3 +1,4 @@
+declare function tonumber(this: void, value: any, base?: number): number | undefined;
```

Comment:
What do you think about adding `/** @noSelfInFile */` to this file instead of the `this: void`s?